docs/usage/dto/1-dto-factory.rst

```diff
@@ -0,0 +1,119 @@
+DTO Factory
+===========
+
+Litestar maintains a suite of DTO factory types that can be used to create DTOs for use with popular data modelling
+libraries, such as ORMs. These take a model type as a generic type argument, and create subtypes of
```

Comment:
```suggestion
libraries, such as SQLAlchemy or Pydantic. These take a model type as a generic type argument, and create subtypes of
```

"ORM" isn't really a popular library so this should either be concrete examples of libraries or simply refer to "data modelling libraries, such as ORMs".